src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -3080,6 +3080,24 @@ public EmitDifferenceResult EmitDifference(
             return diff;
         }
 
+        /// <summary>
+        /// Emit the differences between the compilation and the previous generation
+        /// for Edit and Continue. The differences are expressed as added and changed
+        /// symbols, and are emitted as metadata, IL, and PDB deltas. A representation
+        /// of the current compilation is returned as an EmitBaseline for use in a
+        /// subsequent Edit and Continue.
+        /// </summary>
+        [Obsolete("UpdatedMethods is now part of EmitDifferenceResult, so you should use an overload that doesn't take it.")]
```

Comment:
It doesn't look like this overload accepts UpdatedMethods, so I'm not sure why it's being deprecated.

Reply:
Stale message. Will update.